todo-frontend-react-typescript-feature-override/package.json

```diff
@@ -16,10 +16,15 @@
   },
   "devDependencies": {
     "@types/node": "^12.12.0",
-    "react-scripts-ts": "^3.1.0",
     "@types/react": "^16.9.42",
     "@types/react-dom": "^16.9.8",
-    "featurehub-eventsource-sdk": "^2.0.0",
+    "react-scripts-ts": "^4.0.8",
```

Comment:
have you forgotten the eventsource dependency?

Reply:
i probably had, fixed now